reportportal/templates/api-deployment.yaml

```diff
@@ -15,22 +15,45 @@ spec:
     spec:
       containers:
       - env:
+        - name: "LOGGING_LEVEL_ORG_HIBERNATE_SQL"
+          value: "info"
         - name: "JAVA_OPTS"
-          value: "-Xmx1g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp"
-        - name: "RP_PROFILES"
-          value: "docker"
-        {{ if .Values.mongodb.endpoint.external }}
-        - name: "RP_MONGO_URI"
-          value: "{{ .Values.mongodb.endpoint.address }}:{{ .Values.mongodb.endpoint.port }}"
-        {{- end }}
-        - name: HOSTNAME
+          value: "-Xmx1g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp  -Dcom.sun.management.jmxremote.rmi.port=12349 -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=false  -Dcom.sun.management.jmxremote.port=9010 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=10.6.128.241"
```

Comment:
You can tell the JVM to honour the container limit. “-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap” instead of -Xmx.